esp-hal/src/spi/master.rs

```diff
@@ -50,6 +50,8 @@
 use core::marker::PhantomData;
 
 use fugit::HertzU32;
+#[cfg(feature = "optimize-spi-in-iram")]
```

Comment:
This is the wrong cfg, though CI still passes so maybe we don't need it at all?

Reply:
I think it's because CI is not testing for the feature since it's not default